swan-lake/resources/featured-scenarios/write-a-restful-api-with-ballerina.md

```diff
@@ -301,69 +398,42 @@ public type ErrorMsg record {|
 
 ## Run the service
 
-In the terminal, navigate to the `covid19` directory, and execute the command below to run the service package.
+Use the [**Run**](/learn/vs-code-extension/run-a-program/) CodeLens of the VS Code extension to build and run the service, as shown below.
 
-> **Info**: The console should have warning logs related to the isolatedness of resources. It is a built-in service concurrency safety feature of Ballerina.
+![Run the service](/learn/images/featured-scenarios/write-a-restful-api-with-ballerina/run-the-service.gif)
 
+>**Info:** Alternatively, you can run this service by navigating to the project root (i.e., the `covid19` directory) and executing the `bal run` command. The console should have warning logs related to the isolatedness of resources. It is a built-in service concurrency safety feature of Ballerina.
 
-```
-$ bal run
-```
-
-You view the output below.
+You can view the output below in the Terminal.
 
 ```
 Compiling source
-	example/covid19:0.1.0
+	ballerina/covid19:0.1.0
```

Comment:
Shall we use a different org name? ballerina is reserved for the ballerina platform.

Reply:
Changed back to `example`.